### PR TITLE
App colors

### PR DESCRIPTION
**Pull request by caiogtoledo**

### CODE REVIEW

**Comment by hectorguerrini** [comment]
``` 
theme: ThemeData(
        fontFamily: 'Rubik',
        colorScheme: ColorScheme.light().copyWith(
            primary: AppColors.primary, secondary: AppColors.secondary),
        scaffoldBackgroundColor: Colors.white,
        elevatedButtonTheme: AppTheme.elevatedButtonTheme,
        cardTheme: AppTheme.cardTheme,
      )
```

File: lib/shared/widgets/app_bar_widget.dart [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import 'package:flutter/material.dart';
+import 'package:system_maua_front/shared/themes/app_gradients.dart';
+
+class AppBarWidget extends StatelessWidget implements PreferredSizeWidget {
+  final Text title;
+  final AppBar appBar;
```
Comment (hectorguerrini) [inline lib/shared/widgets/app_bar_widget.dart]: Trocar para 
`final double? toolbarHeight;`

File: lib/shared/widgets/app_bar_widget.dart [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+      flexibleSpace: Container(
+        decoration: BoxDecoration(
+          gradient: AppGradients.blueGradient,
+        ),
+      ),
+    );
+  }
+
+  @override
+  Size get preferredSize => Size.fromHeight(appBar.preferredSize.height);
```
Comment (hectorguerrini) [inline lib/shared/widgets/app_bar_widget.dart]: Size.fromHeight(toolbarHeight ?? kToolbarHeight),

**Review by DanielBranquinho — APPROVED** [review]
Lindo

**Pull request merged**